gfx_scene.h

```diff
@@ -26,6 +26,7 @@ SOFTWARE.
 
 #include "gfx.h"
 #include "glm/glm.hpp"
+#include "glm/gtc/type_ptr.hpp"
```

Comment:
Please move this include to the implementation details section so we try and keep the "public" header as clean from includes as possible.

Reply:
Done.